app/fragments/ContactFragment.tsx

```diff
@@ -49,6 +50,19 @@ export const ContactFragment = fragment(() => {
     const removeContact = useRemoveContact();
     const safeArea = useSafeAreaInsets();
     const contact = useContact(params.address);
+    const contractInfo = useContractInfo(params.address ?? '');
+    const [bounceableFormat,] = useBounceableWalletFormat();
+
+    const addressString = useMemo(() => {
```

Comment:
It is not complete address, address short string maybe?

Reply:
✅